test/conformance/ingress/update_test.go

```diff
@@ -0,0 +1,195 @@
+// +build e2e
```

Comment:
This seems like it belongs in e2e and not conformance...

Reply:
it's a separate directory / package.  It is conformance, just not for public API, but for what it means to be a conformant networking abstraction for knative/serving.

We do need to rationalize our longer-term story for where the networking types and conformance suite live, this is something @tcnghia and I have been discussing now that there are more than a few networking implementations.